src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -87,7 +109,7 @@ public PsGithub(final String gapp, final String gkey) {
             throw new IllegalArgumentException("code is not provided");
         }
         return Collections.singleton(
-            PsGithub.fetch(this.token(href.toString(), code.next()))
+            this.fetch(this.token(href.toString(), code.next()))
```

Comment:
@dmzaytsev this will work with Github, they allow any redirect_uri, as long as it is in the same domain as the one registered with them (to my knowledge). only Google is very strict about redirect_uri